Test/contract-wrappers/AllExterns.dfy.expect

```diff
@@ -3,9 +3,9 @@ CheckExtern.dfy(22,12): Warning: The ensures clause at this location cannot be c
 
 Dafny program verifier did not attempt verification
 FooTest: FAILED
-	CheckExtern.dfy(3,12): Runtime failure of ensures clause from CheckExtern.dfy:3:13
+	CheckExtern.dfy(3,10): Runtime failure of ensures clause from CheckExtern.dfy:3:13
```

Comment:
Previously, the error was pointing correclty on the token that was the most representative of the clause:
```
  ensures y >= 10
            ^^
```
With your change, it now points to the start.
```
  ensures y >= 10
          ^
```

which is much more ambiguous (consider a conjunction `a && b`, one might think that the error comes from a() and not from the entire conjunction). Could you put back the previous behavior please?